src/Core/Components/InputFile/FluentInputFile.razor.js

```diff
@@ -21,3 +21,51 @@ export function previewImage(inputElem, index, imgElem) {
     imgElem.src = url;
     imgElem.alt = inputElem.files[index].name;
 }
+
+export function initializeFileDropZone(containerElement, inputFile) {
+    function onDragHover(e) {
+        e.preventDefault();
+        containerElement.setAttribute("drop-files", "true");
+    }
+
+    function onDragLeave(e) {
+        e.preventDefault();
+        containerElement.removeAttribute("drop-files");
+    }
+
+    // Handle the paste and drop events
+    function onDrop(e) {
+        e.preventDefault();
+        containerElement.removeAttribute("drop-files");
+
+        // Set the files property of the input element and raise the change event
+        inputFile.files = e.dataTransfer.files;
+        const event = new Event('change', { bubbles: true });
+        inputFile.dispatchEvent(event);
+    }
+
+    function onPaste(e) {
+        // Set the files property of the input element and raise the change event
+        inputFile.files = e.clipboardData.files;
+        const event = new Event('change', { bubbles: true });
+        inputFile.dispatchEvent(event);
+    }
+
+    // Register all events
+    containerElement.addEventListener("dragenter", onDragHover);
```

Comment:
I prefer to keep the `div` attributes to avoid to create event listener. What do you think?

Reply:
This is needed to fix the fact that the attributes don't trigger anything when a file is dropped a second tim. With JS this does happen